RedisCore.Tests/RedisClientTests.cs

```diff
@@ -50,6 +50,37 @@ public async Task RedisClient_Get_Set_Delete_Test(RedisClientConfig config)
             }
         }
         
+        [TestMethod]
+        [DynamicData(nameof(Test_Endpoints_Data), typeof(RedisClientTestsBase), DynamicDataSourceType.Method)]
+        public async Task RedisClient_Set_With_Concurrency_Test(RedisClientConfig config)
+        {
+            using (var client = new RedisClient(config))
+            {
+                for (var i = 0; i < 8; ++i)
+                {
+                    var testKey = UniqueString();
+                    var testValue = UniqueString();
+
```

Comment:
Missing corner case?

Trying to delete a set with does not exist:

                    await client.Delete(testKey);
                    Assert.IsNull(await client.GetOrDefault<string>(testKey));

Maybe this does not belong here, since does not test any concurrency flag.

Reply:
We should delete the key to not leave a garbage in the database  but we don't need to check the result since it is tested in different test case